pkg/noderesourcetopology/score.go

```diff
@@ -65,17 +66,20 @@ func (tm *TopologyMatch) Score(ctx context.Context, state *framework.CycleState,
 	nodeTopology := findNodeTopology(nodeName, tm.lister)
 
 	if nodeTopology == nil {
+		klog.V(4).InfoS("noderesourcetopology was not found for node", "node", nodeName)
 		return 0, nil
 	}
 
-	klog.V(5).InfoS("NodeTopology found", "nodeTopology", nodeTopology)
+	if err := logNRT("noderesourcetopology found", nodeTopology); err != nil {
```

Comment:
I think we should not fail because of error here. We probably should log the logging error (!!) and carry on